src/pages/super-admin/SuperAdminEmail.tsx

```diff
@@ -0,0 +1,65 @@
+import SuperAdminSearchBar from '@components/super-admin/workspace/SuperAdminSearchBar';
+import styled from '@emotion/styled';
+import { colFlex } from '@styles/flexStyles';
+import { useEffect, useRef, useState } from 'react';
+import Pagination from '@components/common/pagination/Pagination';
+import AppContainer from '@components/common/container/AppContainer';
+import { useNavigate } from 'react-router-dom';
+import SuperAdminSearchContents from '@components/super-admin/SuperAdminSearchContents';
+import { PaginationResponse, EmailDomain } from '@@types/index';
+import { defaultPaginationValue } from '@@types/PaginationType';
+import useSuperAdminEmail from '@hooks/super-admin/useSuperAdminEmail';
+import SuperAdminEmailContent from '@components/super-admin/user/SuperAdminEmailContent';
+
+const ContentContainer = styled.div<{ justifyCenter?: boolean }>`
+  height: 550px;
+  ${(props) =>
+    colFlex({
+      justify: props.justifyCenter ? 'center' : 'flex-start',
+      align: 'center',
+    })}
+`;
+
+function SuperAdminEmail() {
+  const pageSize = 6;
+  const navigate = useNavigate();
+  const [Emails, setEmails] = useState<PaginationResponse<EmailDomain>>(defaultPaginationValue);
```

Comment:
```suggestion
  const [emails, setEmails] = useState<PaginationResponse<EmailDomain>>(defaultPaginationValue);
```
변수는 camel case로 작성해주세요